_posts/2022-09-27-building-container-images.adoc

```diff
@@ -0,0 +1,173 @@
+---
+title: Building Containers in Rootless Mode on OpenShift
+layout: post
+author: Mario Loriedo
+description: >-
+  In this blog post, we are going to see how to configure Eclispe Che to run `podman build` from a remote development environment.
+categories: []
+keywords: ['buildah', 'podman']
```

Comment:
It may make some sense to include `docker` as a keyword here.

Reply:
Added